AttoBot/lambda_function.py

```diff
@@ -25,6 +25,12 @@
 
 SECRET = os.environ["SECRET"]
 
+TAG_REQ = "\n".join((
+    "Please make sure that:",
+    "- [ ] CI passes for supported Julia versions (if applicable).",
```

Comment:
worth mentioning now that JuliaCIBot will run tests against the package and its reverse dependencies to see if breaking changes are going to cause problems